addons/l10n_es_edi_facturae/models/account_move.py

```diff
@@ -699,3 +707,58 @@ def _search_product_for_import(self, item_description):
         code_and_name = re.match(r"(\[(?P<default_code>.*?)\]\s)?(?P<name>.*)", item_description).groupdict()
         product = self.env['product.product']._retrieve_product(**code_and_name)
         return product
+
+    # -------------------------------------------------------------------------
+    # BUSINESS METHODS                                                        #
+    # -------------------------------------------------------------------------
+    def _sign_xml(self, edi_data, signature_data):
```

Comment:
All methods in this file are wrongly named:
```suggestion
    def _l10n_es_facturae_sign_xml(self, edi_data, signature_data):
```